src/patternfly/components/TreeView/tree-view.scss

```diff
@@ -0,0 +1,189 @@
+$pf-c-tree-view--MaxNesting: 6;
+
+.pf-c-tree-view {
+  --pf-c-tree-view--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-tree-view--PaddingBottom: var(--pf-global--spacer--sm);
+
+  // Link base
+  --pf-c-tree-view__link--base--padding: var(--pf-global--spacer--sm);
+  --pf-c-tree-view__link--padding-nested: var(--pf-global--spacer--lg);
+  --pf-c-tree-view__item--m-expandable__link--padding-nested: var(--pf-global--spacer--lg);
+
+  // Link
+  --pf-c-tree-view__link--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-tree-view__link--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-tree-view__link--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-tree-view__link--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-tree-view__link--Color: var(--pf-global--Color--100);
+  --pf-c-tree-view__link--m-current--Color: var(--pf-global--link--Color);
+  --pf-c-tree-view__link--m-current--FontWeight: var(--pf-global--FontWeight--bold);
+  --pf-c-tree-view__link--m-first--FontWeight: var(--pf-global--FontWeight--bold);
```

Comment:
what do you think about `m-root` instead of `m-first`.

Reply:
yep thats good

Reply:
yeah thats a good idea!